### DISCUSSION TITLE
Should `--etcd-s3` be configured on only one or all etcd nodes?

### DISCUSSION THREAD

**Discussion by thewilli** [k3s-io/k3s#7523] [Q&A]
k3s provides the option `--etcd-s3` which allows to backup etcd snapshots to an S3 bucket.

If I wanted to enable this on my HA k3s cluster, should I add it to all nodes or only a particular one?

 I'm not sure about the behaviour for scheduled backups. If the setting was enabled on all nodes, would all etcd nodes than upload snapshots, so would I end up with lots of redundant snapshots on S3, or would there be a single instance doing the S3 upload at a time?

**Comment by brandond** [comment]
It should be configured on all the nodes that you want to upload snapshot to s3. All configured nodes upload snapshots. Most folks like knowing that their snapshots don't depend on a specific node being up.
